x-pack/plugins/security_solution/public/detections/components/alerts_table/alerts_utility_bar/index.tsx

```diff
@@ -47,6 +47,17 @@ const UtilityBarFlexGroup = styled(EuiFlexGroup)`
   min-width: 175px;
 `;
 
+const BuildingBlockContainer = styled(EuiFlexItem)`
+  background: repeating-linear-gradient(
+    135deg,
```

Comment:
`127deg` here will result in the crosshatch lining up across multiple rows, which seems to be a bit less jarring and easier to focus on the content (n=1 here ;P).

##### `135deg`
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/94869876-74726880-0403-11eb-8183-aef208d07008.png" />
</p>

##### `127deg`
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/94869928-9370fa80-0403-11eb-8045-6a5de85eca46.png" />
</p>